packages/inetutils.rb

```diff
@@ -36,9 +39,36 @@ def self.build
       --enable-authentication \
       --disable-servers"
     system 'make'
+    system "cat <<'EOF'> ping_
+#!/bin/bash
+sudo -E #{CREW_PREFIX}/sbin/capsh --caps=\"cap_net_raw+eip cap_setpcap,cap_setuid,cap_setgid+ep\" \\
+    --keep=1 --user=nobody --addamb=cap_net_raw -- \\
+    -c \"#{CREW_PREFIX}/bin/ping.elf \$@\"
+EOF"
+    system "cat <<'EOF'> ping6_
+#!/bin/bash
+sudo -E #{CREW_PREFIX}/sbin/capsh --caps=\"cap_net_raw+eip cap_setpcap,cap_setuid,cap_setgid+ep\" \\
+    --keep=1 --user=nobody --addamb=cap_net_raw -- \\
+    -c \"#{CREW_PREFIX}/bin/ping6.elf \$@\"
+EOF"
+    system "cat <<'EOF'> traceroute_
+#!/bin/bash
+sudo -E #{CREW_PREFIX}/sbin/capsh --caps=\"cap_net_raw+eip cap_setpcap,cap_setuid,cap_setgid+ep\" \\
+    --keep=1 --user=nobody --addamb=cap_net_raw -- \\
+    -c \"#{CREW_PREFIX}/bin/traceroute.elf \$@\"
```

Comment:
Is sudo necessary?  I'm thinking maybe not?

Reply:
You have to use sudo to elevate permissions to use capsh.

Reply:
Shouldn't need to use sudo to ping.  I'm confused.